native/src/components/SearchInput.tsx

```diff
@@ -20,35 +21,47 @@ const Wrapper = styled.View<{ space: boolean }>`
   background-color: ${props => props.theme.colors.backgroundColor};
 `
 
+const StyledHelperText = styled(Text)`
+  width: fit-content;
+  padding: 0 35px;
```

Comment:
🔧 This is too much, if you change `margin: 0 5px;` to `margin: 0 4px;` above you can do
```suggestion
  padding-left: 28px;
```